src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar main-search-bar"
+      data-qa="big"
+    >
+      <input
+        class="search-bar-input search-bar-input--main"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar additional-search-bar"
+      data-qa="small"
+    >
+      <input
+        class="search-bar-input search-bar-input--additional"
+        type="text"
+        data-qa="hover"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
```

Comment:
Same comment as for the previous `form` tag. If you don't need to collect user input, consider using a `div` or `section` tag.